services/movie.py

```diff
@@ -1,11 +1,15 @@
+from typing import Optional
+
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        genres_ids: list[int] = None,
```

Comment:
The type hint `list[int]` is only supported in Python 3.9 and later. If you need compatibility with earlier Python versions, consider using `List[int]` from the `typing` module.